### PR TITLE
feat(e2e): add ng e2e command

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/e2e/app.e2e.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import "angular2/testing";
+/// <reference path="../typings/main.d.ts" />
```
Comment (hansl) [inline addon/ng2/blueprints/ng2/files/e2e/app.e2e.ts]: Could you do the same as the main project and add a `typings.d.ts` file to this directory that includes this line. Include that new file in the build.

Comment (hansl) [inline addon/ng2/blueprints/ng2/files/e2e/app.e2e.ts]: Actually, scratch that last comment.

**Comment by hansl** [comment]
LGTM.

File: README.md [hunk 2]
```diff
@@ -151,16 +151,13 @@ Tests will execute after a build is executed via [Karma](http://karma-runner.git
 
 ### Running end-to-end tests
 
-Before running the tests make sure that you have an updated webdriver and that
-the tests are built:
-
 ```bash
-$(npm bin)/webdriver-manager update
-$(npm bin)/tsc -p e2e/
+ng test
```
Comment (Brocco) [inline README.md]: The command here should be `ng e2e`

Comment (filipesilva) [inline README.md]: Good catch, thanks!

**Comment by Brocco** [comment]
LGTM

**Pull request merged**